RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,79 @@
+#ifndef RecoTracker_LSTCore_interface_alpaka_Constants_h
+#define RecoTracker_LSTCore_interface_alpaka_Constants_h
+
+#include "RecoTracker/LSTCore/interface/Constants.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
+
+  using namespace ::lst;
```

Comment:
this triggers a code-rule violation
```
Rule1 : https://raw.githubusercontent.com/cms-sw/cmssw/master/Utilities/ReleaseScripts/python/cmsCodeRules/config.py
Search for "using namespace" or "using std::" in header files

/RecoTracker/LSTCore/interface/alpaka/Constants.h
[8]
```

This was suggested to be used during the review in https://github.com/cms-sw/cmssw/pull/45117#discussion_r1724885193
for [RecoTracker/LSTCore/src/alpaka/Event.h](https://github.com/cms-sw/cmssw/pull/45117/files/ff27cf3a74126ec211afe219415f242d7797228f#diff-33b421cab4b2db0608d47e4ffd51355002f52ab455810d7a7f600d7e5e3b6cf9)

Do I understand correctly that the rule is not triggered for header files in src/?

@fwyzard 
What should we do?
- leave as false-positive/acceptable
- move this expression to src/

Please advise.

Reply:
The false positive can be accepted. This line doesn't really violate any code rule (as the `using namespace` doesn't impact the global namespace), but haven't gotten to improve the checker yet. See also https://github.com/cms-sw/cmssw/issues/40322.